### PR TITLE
docs: update signMessageWith6492 arg doc

### PR DESCRIPTION
**Pull request by avasisht23**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR updates the parameter name and type in the `signMessageWith6492.md` file.

### Detailed summary
- Renamed parameter `msg` to `message` of type `SignableMessage`
- Added a link to the message type documentation

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ A Promise containing the signature of the message, additionally wrapped in EIP-6
 
 ## Parameters
 
-### `msg: string | Uint8Array`
+### `message: string | Uint8Array`
```
Comment (moldy530) [inline site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md:44]: I'm not sure the type is correct here either. It should be `SignableMessage` from viem

Comment (denniswon) [reply site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md:44]: yeah, should be `SignableMessage` type for `message` and another nit: return type is set to Promise<Hex> in our code, so would be nice that can be changed as well.

**Review by denniswon — CHANGES_REQUESTED** [review]
https://github.com/alchemyplatform/aa-sdk/pull/560/files#r1566387685

File: site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md (hunk 1 above)
Comment (avasisht23) [reply site/packages/aa-core/smart-account-client/actions/signMessageWith6492.md:44]: yup good catch

**Pull request merged**